ui/src/components/Multi.tsx

```diff
@@ -0,0 +1,35 @@
+import React, { useState } from "react";
+import { MultiSelect } from "react-multi-select-component";
+
+const handleNewField = (value) => ({
+  label: value,
+  value: value.toUpperCase(),
+});
+
+const options = [
+  { label: "Hot", value: "Hot" },
+  { label: "Med", value: "Med" },
+  { label: "Mild", value: "Mild" },
+  { label: "Vegan", value: "Vegan" },
+  { label: "Vegietarian", value: "Vegietarian" },
```

Comment:
I know this is temporarily hard coded but might want to install a vscode package for spell checking. its helped me a bunch